--- boost1.60-1.60.0+dfsg.orig/tools/build/src/tools/gcc.jam
+++ boost1.60-1.60.0+dfsg/tools/build/src/tools/gcc.jam
@@ -1076,7 +1076,6 @@
 # x86 and compatible
 # The 'native' option appeared in gcc 4.2 so we cannot safely use it as default.
 # Use i686 instead for 32-bit.
-toolset.flags gcc OPTIONS <architecture>x86/<address-model>32/<instruction-set> : -march=i686 ;
 cpu-flags gcc OPTIONS : x86 : native : -march=native ;
 cpu-flags gcc OPTIONS : x86 : i486 : -march=i486 ;
 cpu-flags gcc OPTIONS : x86 : i586 : -march=i586 ;
