Index: libtool-2.2.10/tests/deplibs-ident.at
===================================================================
--- libtool-2.2.10.orig/tests/deplibs-ident.at	2010-06-17 18:36:59.000000000 +0000
+++ libtool-2.2.10/tests/deplibs-ident.at	2010-06-17 18:39:32.000000000 +0000
@@ -66,13 +66,6 @@
   AT_CHECK([$LIBTOOL --mode=link $CC $CFLAGS $LDFLAGS -o b$EXEEXT b.$OBJEXT ../liba1.la ../liba2.la ../liba3.la ../../c/libcee.la -rpath /nowhere],
 	   [0],[stdout],[ignore])
   AT_CHECK([$EGREP 'cee.*cee' stdout], 1, [ignore], [ignore])
-  AT_XFAIL_IF([dnl
-    eval `$LIBTOOL --config | $EGREP '^hardcode_(direct|direct_absolute|action)='`
-    case $hardcode_action,$hardcode_direct,$hardcode_direct_absolute in
-      relink,yes,no) :;;
-      *,no,*) :;;
-      *) false;;
-    esac])
   dnl This is currently broken in libtool
 )
 
