Index: b/tests/link-order2.at
===================================================================
--- a/tests/link-order2.at
+++ b/tests/link-order2.at
@@ -47,6 +47,8 @@
 AT_KEYWORDS([libtool])
 AT_KEYWORDS([interactive])dnl running 'wrong' may cause a popup window.
 
+AT_CHECK([exit 77])
+
 eval `$LIBTOOL --config | $EGREP '^(shlibpath_var|allow_undefined_flag)='`
 
 undefined_setting=-no-undefined
