Index: libtool-2.4.6/libtoolize.in
===================================================================
--- libtool-2.4.6.orig/libtoolize.in
+++ libtool-2.4.6/libtoolize.in
@@ -114,7 +114,7 @@ When reporting a bug, please describe a
 include the following information:
 
        host-triplet:   @host_triplet@
-       version:        $progname (GNU @PACKAGE@) @VERSION@
+       version:        $progname $scriptversion
        automake:       `($AUTOMAKE --version) 2>/dev/null |$SED 1q`
        autoconf:       `($AUTOCONF --version) 2>/dev/null |$SED 1q`
 
Index: libtool-2.4.6/build-aux/ltmain.in
===================================================================
--- libtool-2.4.6.orig/build-aux/ltmain.in
+++ libtool-2.4.6/build-aux/ltmain.in
@@ -156,7 +156,7 @@ include the following information:
        compiler:       $LTCC
        compiler flags: $LTCFLAGS
        linker:         $LD (gnu? $with_gnu_ld)
-       version:        $progname (GNU @PACKAGE@) @VERSION@
+       version:        $progname $scriptversion
        automake:       `($AUTOMAKE --version) 2>/dev/null |$SED 1q`
        autoconf:       `($AUTOCONF --version) 2>/dev/null |$SED 1q`
 
